dipy/viz/horizon/tab/base.py

```diff
@@ -38,13 +65,42 @@ def __init__(self, tabs, win_size):
         for id, tab in enumerate(tabs):
             self.__tab_ui.tabs[id].title = ' ' + tab.name
             self.__tab_ui.tabs[id].title_font_size = 18
-            tab.build(id, self.__tab_ui)
+            tab.build(id, self.__tab_ui, self)
 
     def reposition(self, win_size):
         win_width, win_height = win_size
         x_pad = np.rint((win_width - self.__tab_size[0]) / 2)
         self.__tab_ui.position = (x_pad, 5)
 
+    def syncronize_slices(self, active_tab_id, x_value, y_value, z_value):
+        """
+        Synchronize slicers for all the images
```

Comment:
Thank you for fixing the typo. it would be great if you could do it for the function name also. (see just above, typo in `syncronize_slices`)

Reply:
Thanks for pointing out. There were a few more I have fixed all of them.